docs/contribute/writing-articles.md

```diff
@@ -0,0 +1,400 @@
+---
+prev: false
+next: false
+description: How to contribute new articles to the wiki.
+---
+
+# Contributing Articles
+
+Thank you for deciding to contribute articles to this wiki! The only way this wiki has come this far, and for it to continue to grow is through contributions.
+
+This guide will help you write, format, and implement your article into the wiki.
+
+## Forking the GitHub Repository {#forking-the-repo}
+
+In order to send a Pull Request (PR) to the wiki to request your article to be added, you have to fork the [ModdingWiki repository](https://github.com/LethalCompany/ModdingWiki) (repo). 
+
+To fork the wiki, it's fairly simple. You'll click the `Fork` button on the webpage, then click `Create fork`.
+
+![GitHub Fork Repo Button](/public/images/writing-articles/forkgithubrepobutton.png)
+
+![GitHub Create Fork Dialog](/public/images/writing-articles/forkgithubrepopopup.png)
+
+Now we've created the fork! From there you can make changes to a branch on the fork and create the PR when you're finished.
+
+## Creating the File
+
+Creating the article file is pretty simple. Add the file to the folder/subsection you are adding to, such as `/docs/dev/api/` for APIs, and create a markdown file (.md).
+
+::: danger IMPORTANT
+Make sure you make your file name with no spaces or symbols, other than `-` to replace spaces!
+
+For example, this article is named `writing-articles.md` and is located in `/docs/extras/`.
+:::
+
+The terminal will then display the following, at which point you can go to the URL given in the terminal:
+
+![Displayed Text in Terminal](/public/images/writing-articles/runvitepressterminal.png)
```

Comment:
i think this accidentally got put in the wrong section, it should be in `Viewing Your Article`